impeller/compiler/reflector.cc

```diff
@@ -767,6 +773,75 @@ std::vector<StructMember> Reflector::ReadStructMembers(
       continue;
     }
 
+    // Tightly packed half Point (vec2).
+    if (member.basetype == spirv_cross::SPIRType::BaseType::Half &&  //
+        member.width == sizeof(float) * 4 &&                         //
```

Comment:
Does all of the changes in this file have a test?

Reply:
I added a test shader that specifically ensures that we can compile these values. Otherwise these are build changes.